creator-node/src/diskManager.ts

```diff
@@ -625,15 +633,28 @@ async function _migrateFileWithCustomStoragePath(
   )
 
   if (error) {
+    // If copying errored because the file is delisted, we can ignore the error and update the db storagePath
+    const isServable = await serviceRegistry.blacklistManager.isServable(
+      fileRecord.multihash
+    )
+    if (!isServable && fileRecord.skipped) {
+      await models.File.update(
+        { storagePath },
+        {
+          where: { fileUUID: fileRecord.fileUUID }
+        }
+      )
+      return true
+    }
     logErrorWithDuration(
       { logger, startTime: fetchStartTime },
       `Error fixing fileRecord ${JSON.stringify(fileRecord)}: ${error}`
     )
     return false
   } else {
-    // Update file's storagePath in DB to newly saved location, and ensure it's not marked as skipped
+    // Update file's storagePath in DB to newly saved location
     await models.File.update(
-      { storagePath, skipped: false },
+      { storagePath },
```

Comment:
I thought about this more and am actually not sure we want to change skipped to false just because it migrated. what if the track was skipped because it was blacklisted, but it happened to exist at the legacy path and copied over successfully? we probably shouldn't unmark it as skipped, right?

Reply:
imo if the content is available locally theres no reason to mark it as skipped

Reply:
fair enough. BlacklistManager would prevent it from being served anyway